libs/assets/batch/startup.bat

```diff
@@ -0,0 +1,8 @@
+pm2 status | find /i "Current process list running is not in sync with saved list. App effluent-api dispatch-api recycling-api operations-api differs. Type 'pm2 save' to synchronize."
```

Comment:
Process list is not the same for all hosts

Reply:
Removed host specific process list refrence.